home_work_visitor/TransactionVisitor.h

```diff
@@ -0,0 +1,38 @@
+//
+// Created by flyon21 on 09.11.24.
+//
+
+#ifndef C_PRO_TRANSACTIONVISITOR_H
+#define C_PRO_TRANSACTIONVISITOR_H
```

Comment:
Good practic for comercial code:
In such guards we can add gui:
#ifndef C_PRO_TRANSACTIONVISITOR_H_D1978696_E462_4C36_981F_E374D94C6916
#define C_PRO_TRANSACTIONVISITOR_H_D1978696_E462_4C36_981F_E374D94C6916

This is even more safe